src/__snapshots__/index.test.ts.snap

```diff
@@ -66,6 +66,9 @@ exports[`outputted Android matches snapshot 1`] = `
 exports[`outputted CommonJS matches snapshot 1`] = `
 "\\"use strict\\";
 Object.defineProperty(exports, \\"__esModule\\", { value: true });
+exports.tpColorRed600 = exports.tpColorRed500 = exports.tpColorRed = exports.tpColorRed300 = exports.tpColorRed200 = exports.tpColorRed100 = exports.tpColorYellow600 = exports.tpColorYellow500 = exports.tpColorYellow = exports.tpColorYellow300 = exports.tpColorYellow200 = exports.tpColorYellow100 = exports.tpColorGreen600 = exports.tpColorGreen500 = exports.tpColorGreen = exports.tpColorGreen300 = exports.tpColorGreen200 = exports.tpColorGreen100 = exports.tpColorPurple600 = exports.tpColorPurple500 = exports.tpColorPurple = exports.tpColorPurple300 = exports.tpColorPurple200 = exports.tpColorPurple100 = exports.tpColorIndigo600 = exports.tpColorIndigo500 = exports.tpColorIndigo = exports.tpColorIndigo300 = exports.tpColorIndigo200 = exports.tpColorIndigo100 = exports.tpColorBlue600 = exports.tpColorBlue500 = exports.tpColorBlue = exports.tpColorBlue300 = exports.tpColorBlue200 = exports.tpColorBlue100 = exports.tpBreakpointSplitViewSmallValue = exports.tpBreakpointSplitViewSmall = exports.tpBreakpointSplitViewMediumValue = exports.tpBreakpointSplitViewMedium = exports.tpBreakpointSmallValue = exports.tpBreakpointSmall = exports.tpBreakpointMediumValue = exports.tpBreakpointMedium = exports.tpBreakpointLargeValue = exports.tpBreakpointLarge = exports.tpBorderRadiusSides = exports.tpBorderRadiusFull = exports.tpBorderRadiusBig = exports.tpBorderRadiusBase = void 0;
```

Comment:
This seems incorrect, I think?

Reply:
hmm actually i think this is just an artifact of the TS version upgrade. all these values are still assigned correctly and exported below. it just also has this boilerplate of initialising them up here.

Reply:
Weird!